### PR TITLE
[MOD-197][Improvement] actions --> reviewActions

### PR DESCRIPTION
**Pull request by laurenbarker**
*The ember-osf-reviews changes rely on the API changes and ember-osf changes in the related tickets below.*

## Purpose
Change `actions` --> `reviewActions` to support generalizing the actions API.

Related tickets: 
* https://github.com/CenterForOpenScience/osf.io/pull/7924 but with updated changes https://github.com/mfraezz/osf.io/tree/feature/request-actions
* https://github.com/CenterForOpenScience/ember-osf/pull/329
    * will need to be merged into our version of ember-osf (they aren't merged quite yet)

## Changes
* mainly just update the name
* actions for the actions-list are loaded from `reviews/actions` instead of the user relationship

## QA notes
* regression for moderation functionality; everything should work the same

### CODE REVIEW

**Comment by aaxelb** [comment]
@laurenbarker Would you mind setting the ember-osf dependency to that PR's branch, so travis can actually run? We'll have to update the dependency anyway once those changes are released.

File: app/components/action-feed/component.js [hunk 1]
```diff
@@ -41,18 +40,18 @@ export default Component.extend({
...
+            const actions = yield this.get('store').query('review-action', {
                 page,
-                embed: 'target',
+                embed: ['target'],
             });
             this.get('actionsList').pushObjects(actions.toArray());
             this.set(
                 'totalPages',
-                Math.ceil(actions.get('meta.total') / actions.get('meta.per_page')),
+                Math.ceil(actions.get('meta.pagination.total') / actions.get('meta.pagination.per_page')),
```
Comment (aaxelb) [inline app/components/action-feed/component.js:50]: Which version of the API uses `meta.pagination`? Is that new?

**Review by aaxelb — COMMENTED** [review]
Looks good, just one question

File: app/components/action-feed/component.js (hunk 1 above)
Comment (laurenbarker) [reply app/components/action-feed/component.js:50]: That is a great question. When I was testing it out `meta.total` was incorrect but `meta.pagination.total` was an accurate number. I'll look into why it showed up.

Comment (laurenbarker) [reply app/components/action-feed/component.js:50]: https://github.com/CenterForOpenScience/ember-osf/blob/develop/addon/serializers/osf-serializer.js#L143 (h/t @jamescdavis)

Comment (jamescdavis) [reply app/components/action-feed/component.js:50]: `meta.total` is actually total pages. It should probably be renamed to `total_pages` so this is clear.

Comment (aaxelb) [reply app/components/action-feed/component.js:50]: Ok, I think this is fine for now. If/when `total_pages` is available, we should use that instead.

**Review by aaxelb — APPROVED** [review]

**Pull request merged**